src/freenas/root/.zshrc

```diff
@@ -1,9 +1,15 @@
 
 HISTFILE=~/.zsh-histfile
 SAVEHIST=1000
+PROMPT="%n@%m[%40<...<%~%<<]# "
 
 setopt APPEND_HISTORY
 
+bindkey "^[[F" end-of-line
+bindkey "^[[H" beginning-of-line
+bindkey "^[[3~" delete-char
+bindkey '^H' backward-delete-word
+
 # Enable the builtin emacs(1) command line editor in sh(1),
 # e.g. C-a -> beginning-of-line.
 set -o emacs
```

Comment:
Is this necessary? Not everyone uses emacs or wants emacs key bindings :-/

Reply:
I dont use emacs either, but thats the default of zsh. We cant have both ways. Do you prefer we dont have any keys by default?

Reply:
Disregard. I looked at default .zshrc. Of course key bindings are useful, I just didn't think we want to force anyone one way or the other, but it's easy enough to switch.